redwood/api/src/functions/getVerifiedExternalAddresses/getVerifiedExternalAddresses.ts

```diff
@@ -15,7 +16,7 @@ export const handler: Handler = async (event) => {
   const connections = await db.connection.findMany({
     where: {
       purposeIdentifier,
-      externalAddress: {in: externalAddresses.map((str) => str.toLowerCase())},
```

Comment:
@RoboTeddy we want to standardize on using checksum formatted addresses everywhere, right? I'm planning on adding a new GraphQL directive for all the Ethereum address fields that'll verify that on the way in, but just came across this place where that would break.

I'm going to add a dire

Reply:
Yup do want to standardize on checksum-formatted addresses everywhere.

In terms of what we accept over API calls — we may want to follow [Postel's Law](https://en.wikipedia.org/wiki/Robustness_principle) by accepting addresses in any state of capitalization, but only store/transmit properly-formatted ones